nucypher/characters/lawful.py

```diff
@@ -1221,7 +1221,7 @@ def __bytes__(self):
 
         version = self.TEACHER_VERSION.to_bytes(2, "big")
         interface_info = VariableLengthBytestring(bytes(self.rest_interface))
-        decentralized_identity_evidence = VariableLengthBytestring(self.decentralized_identity_evidence)
+        decentralized_identity_evidence = VariableLengthBytestring(self.decentralized_identity_evidence)  # TODO: Change to fixed length
```

Comment:
To be clear, I didn't change this here to a fixed length field, since it wouldn't be interoperable anymore with other nodes' DiscoveryLoop in IBEX, and I'd like to have consensus in that kind of changes. I'm particular interested in what @jMyles this of this one. Perhaps it's a good chance to test incrementing the teacher version.